assets/0017-SequenceDiagram-InvitationPhase.puml

```diff
@@ -0,0 +1,52 @@
+' DIAGRAM #########################################
+' Invitation phase messaging
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Maker as M
+participant "Social\nMedia" as I
```

Comment:
I think we can use a generic term here: `Marketplace` and not restrict it to social medias.